2.cellprofiler_analysis/NF1_analysis_3channel.cppipe

```diff
@@ -196,7 +207,7 @@ MeasureObjectNeighbors:[module_num:13|svn_version:'Unknown'|variable_revision_nu
     Name the output image:PercentTouching
     Select colormap:Oranges
 
-MeasureObjectIntensityDistribution:[module_num:14|svn_version:'Unknown'|variable_revision_number:6|show_window:False|notes:['Measure object intensity distributions witin objects using center x,y coords for each single object.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+MeasureObjectIntensityDistribution:[module_num:14|svn_version:'Unknown'|variable_revision_number:6|show_window:False|notes:['Measure object intensity distributions witin objects using center x,y coords for each single object.', '', 'Currently we do not measure the Zernikes.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```

Comment:
Can you remind me why we're not measuring zernike features?

Reply:
I think I did this due to computational time: 
![image](https://github.com/WayScience/nf1_cellpainting_data/assets/107513215/414bc98b-fca1-4ae1-853d-a35bd63f8ce8)

@gwaybio @MattsonCam 

Should I rerun the analysis and collect the zernikes? It took about 28 hours to run all of the plates with 5 workers, so it would take day to get analysis (or less depending on if adding all the workers helps).